src/_guides/language/evolution.md

```diff
@@ -273,13 +273,23 @@ other comments can appear before the `@dart` comment.
 For more information about how language versioning works, see the
 [language versioning specification][language versioning feature].
 
+### Dart 2.12
+
+Dart 2.12 added support for **[sound null safety][]**.
+When you opt into null safety, types in your code are non-nullable by default,
+meaning that variables can’t contain null unless you say they can.
+With null safety, your runtime null-dereference errors
+turn into edit-time analysis errors.
+
+In Dart 2.12, **[Dart FFI][]** graduated from beta to the stable channel.
 
 [2.8 breaking changes]: https://github.com/dart-lang/sdk/issues/40686
 [calling native C code]: /guides/libraries/c-interop
 [collection for]: /guides/language/language-tour#collection-operators
 [collection if]: /guides/language/language-tour#collection-operators
 [Dart library]: /guides/libraries/create-library-packages#organizing-a-library-package
 [`dart compile`]: /tools/dart-compile
+[DArt FFI]: /guides/libraries/c-interop
```

Comment:
I don't think this will actually cause an issue with the linking but `DArt` -> `Dart`